src/main/java/ee/carlrobert/codegpt/toolwindow/chat/ui/textarea/ModelComboBoxAction.java

```diff
@@ -87,8 +94,16 @@ public JComponent createCustomComponent(
         getLlamaCppPresentationText(),
         Icons.Llama,
         presentation));
-    actionGroup.addSeparator();
-    actionGroup.add(createModelAction(ServiceType.YOU, "You.com", Icons.YouSmall, presentation));
+    actionGroup.addSeparator("You.com");
+    List.of(
```

Comment:
Could we render these new modes/models only when the user has an active You.com session? Otherwise, they will occupy too much space in the popup.

Reply:
They blocked by account while I tested it by logging in and out, but done 🤣

Reply:
I had the same problem during the initial implementation 😄